plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -247,7 +247,7 @@ function wc_set_loop_prop( $prop, $value = '' ) {
  *
  * @since 4.4.0
  * @param int  $product_id Product it to cache visbiility for.
```

Comment:
This can also use correction
```suggestion
 * @param int  $product_id Product it to cache visibility for.
```